test/server/cards/06-WoE/Grondal.spec.js

```diff
@@ -0,0 +1,37 @@
+describe('Grondal', function () {
+    describe("Grondal's ability", function () {
+        beforeEach(function () {
+            this.setupTest({
+                player1: {
+                    house: 'mars',
+                    token: 'grumpus',
+                    hand: ['grondal'],
+                    deck: ['toad', 'toad']
+                },
+                player2: {
+                    inPlay: ['urchin', 'dust-pixie']
+                }
+            });
+        });
+
+        it('makes two tokens on play', function () {
+            this.player1.playCreature(this.grondal);
+            this.player1.clickPrompt('Left');
+            //this.player1.clickPrompt('Left');
```

Comment:
@romanogit it's not prompting left/right twice when making two tokens.  There might be a bug there but I'm not sure and haven't had time to look into it yet.  It definitely makes two tokens so I'm not sure why it's not prompting for the second one here.

Reply:
Rebased and fixed.